Test/NativeUnitTest/NativeUnitTest.cpp

```diff
@@ -12,12 +12,9 @@ namespace NativeUnitTestForMain
 		
 		TEST_METHOD(TestIfPathExists)
 		{
-			int charOccurs[possibleChars];
-			for (int i = 0; i < possibleChars; ++i) {
-				charOccurs[i] = 0;
-			}
-			Assert::IsFalse(calculateOccursInFile("randompath.txt", {}));
-			Assert::IsTrue(calculateOccursInFile("D:/bruhmoment/Abyss/ToDo.txt", charOccurs));
+			int charOccurs[possibleChars] = {};
```

Comment:
possibleChars should be public constant in some external class. And you have to include it's header

Reply:
It is in a PRAC-12 PR, but I'll probably still make some changes